tests/develop.py

```diff
@@ -29,5 +23,5 @@ def pip_install(*args):
 
 
 if __name__ == '__main__':
-    call('virtualenv', '--distribute', '-p', sys.executable, VENV)
+    call('virtualenv', '-p', sys.executable, VENV)
```

Comment:
Now that Py2 is in the rearview mirror, why not just use the builtin https://docs.python.org/3/library/venv.html ?